.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Persona (link)
+
+Who is the intended audience impacted by this change? This should be a link to a pre-defined persona and not free text
+
+## Motivation/Goal
+
+What is the intention of this change? What outcome do you hope to see from this change?
+
+- Detailed feature1 implemented in this pull request
+- Detailed feature2 implemented in this pull request
+
+## UI changes
+
+**Page name here:**
+**Before:**
+`<ScreenShots>`
+**After:**
+`<ScreenShots>`
+
+## Supporting Evidence (qual or quant)
+
+UI changes should be supported by data and not guesses. What empirical evidence from the above persona do you have to support this change? (usability testing data, interviews, analytics data, a direct quotation from user feedback)
+
+## Browsers Tested
+
+- [ ] Chrome (_version_)
```

Comment:
🤔 until we automate this maybe we make this a table 

>we can change the breakpoints to whatever we have set in semantic or what we find once we get analytics up and running 

 | Browser     |  1024px  | 768px  |  480px | 320px |
| ------------- |:-------------:| -----:|-----:|-----:|
| Browser (_version_)   |  ✅   |   |   |   | |   | 


@dankolbman thoughts?